README.md

```diff
@@ -39,8 +37,6 @@ Once that is saved, you can install the package:
 
 ```shell
 yarn add @yes-theory-fam/database
-# or using npm
-npm install --save @yes-theory-fam/database
```

Comment:
On second thought: This could be left in, as this is documentation about how this package can be used from another project, in which case both yarn and npm could be used.

Reply:
Okay then no need to merge this PR, closing